metacat-common-server/src/main/java/com/netflix/metacat/common/server/util/JdbcUtil.java

```diff
@@ -0,0 +1,54 @@
+/*
+ *       Copyright 2017 Netflix, Inc.
+ *          Licensed under the Apache License, Version 2.0 (the "License");
+ *          you may not use this file except in compliance with the License.
+ *          You may obtain a copy of the License at
+ *              http://www.apache.org/licenses/LICENSE-2.0
+ *          Unless required by applicable law or agreed to in writing, software
+ *          distributed under the License is distributed on an "AS IS" BASIS,
+ *          WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *          See the License for the specific language governing permissions and
+ *          limitations under the License.
+ */
+
+package com.netflix.metacat.common.server.util;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.datasource.DataSourceTransactionManager;
+import org.springframework.transaction.support.TransactionTemplate;
+
+import javax.annotation.Nullable;
+import javax.sql.DataSource;
+
+/**
+ * Jdbc Util.
+ *
+ * @author zhenl
+ * @since 1.1.0
+ */
+public class JdbcUtil {
+    @Getter
```

Comment:
Think we don't need the other stuff generated by @Data, such as equalTo, toString. We might need them later, I will add the @Data annotation.